grails-app/services/org/pih/warehouse/inventory/ProductAvailabilityService.groovy

```diff
@@ -496,7 +497,7 @@ class ProductAvailabilityService {
 						where pa.location = :location
 						${categoriesQuery}
 						group by pa.product
-						""", queryArguments)
+						""".toString(), queryArguments)
```

Comment:
nice catch

Reply:
I described the [here](https://pihemr.atlassian.net/browse/OBGM-491?atlOrigin=eyJpIjoiYjUwN2M1ZTZiNjg3NDg1NDg2MWFmODFmNTI3MzAxYjUiLCJwIjoiaiJ9)